docker-compose.yml

```diff
@@ -69,8 +70,38 @@ services:
       - "./Letterbook.Dashboards/Prometheus/prometheus.yml:/etc/prometheus/prometheus.yml"
     restart: unless-stopped
     
+  tempo:
+    container_name: tempo
+    image: grafana/tempo:2.4.1
+    command: [ "-config.file=/etc/tempo.yaml" ]
+    volumes:
+      - ./Letterbook.Dashboards/Tempo/tempo.yaml:/etc/tempo.yaml
+      - letterbook_tempo_data:/tmp/tempo
+    ports:
```

Comment:
Curious about the ports being used here. We aren't using all of these right? Is this just being done for completeness?

Reply:
I just lifted the whole config from the examples in the tempo github. We probably only need the otlp port for ourselves